wave_vision/tests/fast_tests.cpp

```diff
@@ -0,0 +1,115 @@
+// Libwave Headers
+#include "wave/wave_test.hpp"
+#include "wave/vision/fast_detector.hpp"
+
+namespace wave {
+
+#define TEST_CONFIG "tests/config/fast.yaml"
+#define TEST_IMAGE "tests/data/lenna.png"
+
+// Test fixture to load same image data
+class FASTTest : public testing::Test {
+ protected:
+    FASTTest() {}
+    virtual ~FASTTest() {
+        if (this->detector) {
+            delete this->detector;
+        }
+    }
+
+    void initDetector() {
+        this->detector = new FASTDetector(TEST_CONFIG);
```

Comment:
This was just for the sake of not having to load the figure over and over again. I ended up only using it for one test, but was trying to follow [this](https://github.com/google/googletest/blob/master/googletest/docs/Primer.md#test-fixtures-using-the-same-data-configuration-for-multiple-tests). 

Think I may have just done it wrong...haha.